gradle/dependencies.gradle

```diff
@@ -17,26 +17,31 @@
 def versions = [
         androidTest: '0.5',
         androidxVersion: '1.0.0',
-        uberJava: '0.8.0',
+        uberJava: '0.8.1',
+]
+
+def network = [
+        moshi: 'com.squareup.moshi:moshi:1.9.2',
```

Comment:
We're using GSON inherited from the Java lib right? We shouldn't intermix two json frameworks in an SDK for third parties. We need to be very cautious with transitive deps. I prefer Moshi over GSON, but since we already have GSON we should use that, or move it all to Moshi.

Reply:
I had to add it due to the dependency needed by java sdk. Removed LoginPARDispatcher so don't need this dependency now